samples/databricks-cicd/README.md

```diff
@@ -0,0 +1,70 @@
+# Overview
+
+__What does this sample demonstrate:__
+
+This sample demonstrates how to run unit tests on Python code using [Databricks Connect](https://docs.microsoft.com/en-us/azure/databricks/dev-tools/databricks-connect), run integration tests on Databricks notebooks, and deploy a Databricks job, all in [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions).
+
+This sample contains a single notebook [main_notebook.py](src/main_notebook.py) which calls [some_function](src/dbcicdlib/some_func.py) in a Python module. We run [unit test on some_function](test/unittests/test_some_func.py) and [integration test on main_notebook.py](test/run_notebook_tests.sh).
```

Comment:
Just curious, the 'notebook' file looks more like a conventional python code than a real notebook format? Are there no differences in execution if I would use a `ipynb` file type?

Reply:
Databricks supports [multiple formats](https://docs.microsoft.com/en-us/azure/databricks/notebooks/notebooks-manage#--notebook-external-formats) that it can export to or import from as notebooks. Both Jupyter notebook format `ipynb` and Python are supported. 

When you create a job, you can also [use a Python script or a notebook](https://docs.microsoft.com/en-us/azure/databricks/jobs#--create-a-job). If you use a notebook, it can't be a raw ipynb sitting on dbfs for example, it has to be already imported as a Databricks notebook.

Reply:
Every day something new to learn :) Thanks!